modules/hftt/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/HftT",
+	"url": "https://www.dndbeyond.com/sources/dnd/hftt",
```

Comment:
URL is valid, redirects to `https://www.dndbeyond.com/sources/dnd/hftt/hunt-for-the-thessalhydra`